react/components/SearchResultContainer.js

```diff
@@ -105,6 +105,9 @@ export default class SearchResultContainer extends Component {
         },
       }
     )
+    // Need to redo this to calculate the rest and map in the pageProps
+    restParam && (pageProps.query.rest = restParam)
+    mapParam && (pageProps.query.map = mapParam)
```

Comment:
not going to pass

Reply:
It is a moment solution. Later we could reformat this part to use the value from pageProps.

Reply:
@theomoura but this solution isn't working for all the cases, the only function that should mutate the `map` and `rest` is the `getPagesArgs`

Reply:
I commited a solution for search and not search cases